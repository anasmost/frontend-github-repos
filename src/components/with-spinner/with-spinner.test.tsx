import { shallow } from "enzyme";
import WithSpinner from "./with-spinner";

import React from "react";

describe("<WithSpiner/> Test", () => {
  it("renders WithSpinner", () => {
    const mockProps = {
      isLoading: true,
      children: [],
    };
    expect.assertions(1);
    expect(shallow(<WithSpinner {...mockProps} />)).toMatchSnapshot();
  });
});
