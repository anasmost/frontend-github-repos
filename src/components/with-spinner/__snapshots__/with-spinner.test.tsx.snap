// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WithSpiner/> Test renders WithSpinner 1`] = `ShallowWrapper {}`;
